src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +27,21 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object ob) {
+        Rectangle r = (Rectangle)ob;
```

Comment:
@kenu21 what if ob == null?
What if ob is not a rectangle?
What if you have received the same object?